ml/notebooks/README.md

```diff
@@ -0,0 +1,5 @@
+## Machine Learning Notebooks
+
+This folder is a home for any documentation supporting machine learning related notebooks.
+
+- [Netdata anomaly detection deepdive](netdata%20anomaly%20detection%20deepdive.ipynb): This is a starter notebook to help users understand how anomaly detection works in the Netdata agent and go a little deeper if they want.
```

Comment:
@andrewm4894 I think this `netdata%20anomaly%20detection%20deepdive.ipynb` should be updated to use underscores as well.

Reply:
Good catch, will do.